includes/options/class-amp-options-manager.php

```diff
@@ -17,38 +17,19 @@ class AMP_Options_Manager {
 	 */
 	const OPTION_NAME = 'amp-options';
 
-	/**
-	 * Slug for website experience.
-	 */
-	const WEBSITE_EXPERIENCE = 'website';
-
-	/**
-	 * Slug for stories experience.
-	 *
-	 * @var
-	 */
-	const STORIES_EXPERIENCE = 'stories';
-
 	/**
 	 * Default option values.
 	 *
 	 * @var array
 	 */
 	protected static $defaults = [
-		'experiences'             => [ self::WEBSITE_EXPERIENCE ],
```

Comment:
Removed the `experiences` setting; see https://github.com/ampproject/amp-wp/pull/4315/commits/d24b79c827aa5776eafc84a24d4722aa36e4cdc2.